### PR TITLE
ipython: add page

### PR DESCRIPTION
**Pull request by j86989**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ipython.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# IPython
+
+> A Python shell with automatic history, dynamic object introspection, easier configuration, command completion, access to the system shell and more.
+> More information: https://ipython.org/documentation.html.
+
+- Start an interactive IPython session:
+
+`ipython`
+
+- Enter an interactive IPython session after running the Python script:
```
Comment (zdroid) [inline pages/common/ipython.md]:
```suggestion
- Enter an interactive IPython session after running a Python script:
```

**Comment by sbrl** [comment]
Hey! Next time, you don't have to open a new PR @j86989. You can just push new commits to the branch, and this PR will be updated automatically :-)

**Comment by zdroid** [comment]
... and we'll just squash your commits in the end of the day. It doesn't make a problem if you make one more commit to fix something.

File: pages/common/ipython.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`ipython -i {{script.py}}`
+
+- Create default IPython profile:
+
+`ipython profile create`
+
+- Print the path to the directory for default IPython profile:
```
Comment (agnivade) [inline pages/common/ipython.md]: for _the_ default IPython profile

**Pull request merged**